packages/lesmis/src/api/queries/useVitalsData.js

```diff
@@ -0,0 +1,108 @@
+/*
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ *
+ */
+import { useQuery } from 'react-query';
+import { get } from '../api';
+import { useEntityData } from './useEntityData';
+import { useEntitiesData } from './useEntitiesData';
+
+const getDecendantsOfType = (entities, rootEntityCode, type) => {
+  const entity = entities.find(e => e.code === rootEntityCode);
+
+  if (!entity) {
+    return [];
+  }
+
+  if (entity.type === type) {
+    return [entity.code];
+  }
+
+  if (!entity.childCodes) {
+    return [];
+  }
+  return entity.childCodes.map(c => getDecendantsOfType(entities, c, type)).flat();
+};
+
+const useSchoolInformation = entityCode => {
+  return useQuery(
+    ['vitals', 'school', entityCode],
+    () =>
+      get(`reportData/${entityCode}/LESMIS_school_vitals`, {
+        params: { endDate: '2021-01-01' },
+      }),
+    {
+      staleTime: 1000 * 60 * 60 * 1,
+      refetchOnWindowFocus: false,
+      retry: 1,
+      enabled: true,
+    },
+  );
+};
+
+const useDecendantSchoolInformation = (entities, entityCode) => {
+  const decendants = getDecendantsOfType(entities, entityCode, 'school');
+  return useQuery(
+    ['vitals', 'multi_school', entityCode],
```

Comment:
```suggestion
    ['vitals', 'multiSchool', entityCode],
```

Apart from the rare exception (eg. global constants), it is conventional to use camelCase everywhere in the front end